pkg/sql/parser/sql.y

```diff
@@ -1644,19 +1625,19 @@ import_data_format:
 //
 // %SeeAlso: CREATE TABLE
 import_stmt:
- IMPORT import_bundle_format '(' string_or_placeholder ')'
+ IMPORT import_format '(' string_or_placeholder ')'
   {
     $$.val = &tree.Import{Bundle: true, FileFormat: $2, Files: tree.Exprs{$4.expr()}}
   }
-| IMPORT TABLE table_name FROM import_bundle_format '(' string_or_placeholder ')'
```

Comment:
nit: there's at least one spot in `import_stmt.go` where we say that they parser won't let a non-bundle format show up with`importStmt.Bundle == true` which is now possible

Reply:
Found and removed.